cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -1681,6 +1681,48 @@ bool SubstraitToVeloxPlanConverter::canPushdownOr(
   return true;
 }
 
+bool SubstraitToVeloxPlanConverter::isPushdownSupported(TypePtr inputType) {
+  // keep the same with mapToFilters
+  if (inputType->isDate()) {
+    return true;
+  }
+  switch (inputType->kind()) {
+    case TypeKind::TINYINT:
+    case TypeKind::SMALLINT:
+    case TypeKind::INTEGER:
+    case TypeKind::BIGINT:
+    case TypeKind::REAL:
+    case TypeKind::DOUBLE:
+    case TypeKind::BOOLEAN:
+    case TypeKind::VARCHAR:
+    case TypeKind::ARRAY:
+    case TypeKind::MAP:
```

Comment:
It looks like most type are supported. Would you list the unsupported types instead?